index.html

```diff
@@ -0,0 +1,328 @@
+                    rem voluptatum? Accusantium dolorem dolorum enim maiores maxime nihil nostrum, repudiandae soluta
+                    voluptas voluptatum! Blanditiis, quod?
+                </div>
+            </article>
+            <article class="w-33 big-height light">
+                <h3>Article Section</h3>
+                <div>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid amet inventore minus molestias
+                    nam nisi optio perferendis, quibusdam tenetur vero. Cupiditate eligendi esse, illo natus odit
+                    praesentium suscipit veritatis voluptates.
+                </div>
+            </article>
+            <article class="w-33 big-height light">
+                <h3>Article Section</h3>
+                <div>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A commodi consequuntur dolor eaque,
+                    exercitationem ipsam necessitatibus possimus temporibus voluptas voluptatem? Ab animi cum dolorem
+                    perspiciatis quam, quibusdam quo recusandae tenetur!
+                </div>
+            </article>
+            <article class="w-33 xb-height light padding-40">
+                <h3>Article Section</h3>
+                <div class="rm-padding">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab deleniti illo laborum neque reiciendis
+                    sint vel. Atque consectetur et impedit nam necessitatibus nesciunt perspiciatis placeat quasi
+                    ratione tempora, vitae voluptate?<br>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Asperiores debitis dignissimos dolores
+                    exercitationem facilis, id illo in minus pariatur perspiciatis quasi quibusdam quidem repellat
+                    repudiandae saepe sed sit ullam vel!
+                </div>
+            </article>
+            <article class="w-66 dark xb-height padding-100">
+                <h2>Article Section</h2>
+                <div>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab aut, beatae consequatur corporis eos
+                    fugiat harum, impedit ipsam, laboriosam magnam natus odio officiis quidem quisquam reiciendis
+                    repellendus reprehenderit tempore voluptatum.
+                </div>
+                <div>Ad adipisci aliquam aspernatur doloribus eius fugiat illum iste laboriosam mollitia nemo nesciunt
+                    quae tempora, ut. Beatae blanditiis delectus deleniti id iure, magnam pariatur porro saepe sint,
+                    tempora velit vero.
+                </div>
+            </article>
+        </div>
+        <div class="container">
+            <a href="" class="block w-100 light sm-height">Lorem ipsum dolor sit amet.</a>
+        </div>
+    </main>
+
+    <!--     FOOTER         -->
+    <footer class="dark big-height w-100 padding-100">
+        <h4>Footer Section</h4>
```

Comment:
Let's wrap the footer content with anchor tags `<a>`.

 - [ ]  It contains centered links (content is not important).

Reply:
implemented by @silverside3000